test/unit/lib/exceptionalTimesMapper.js

```diff
@@ -21,11 +22,13 @@ describe('exceptionalOpeningTimesMapper', () => {
     });
 
     it('should populate alterations for in date exceptional opening times', () => {
```

Comment:
Could this be more explicit, i.e. something like 'should populate alterations for exceptional opening times within the next two weeks'